stetho/src/main/java/com/facebook/stetho/inspector/protocol/module/DOM.java

```diff
@@ -226,18 +190,12 @@ public PerformSearchResponse performSearch(JsonRpcPeer peer, final JSONObject pa
         params,
         PerformSearchRequest.class);
 
-    final Pattern queryPattern = Pattern.compile(Pattern.quote(request.query), Pattern.CASE_INSENSITIVE);
+    final ArrayListAccumulator<Integer> results = new ArrayListAccumulator<>();
```

Comment:
I think there's an opportunity to increase clarity here ... what do the integers inside of `results` represent? Are they node IDs? If so, calling this `nodeIdResults` might help to illuminate this fact.